src/components/Subpages/Variants/Variants.tsx

```diff
@@ -125,14 +126,48 @@ export const Variants: React.FC = () => {
         f => featureFilter.indexOf(f) === -1
     )
 
+    async function uploadBlobToGoogleDrive(blob: Blob, fileName: string): Promise<void> {
+
+        const accessToken = '';
+    
+        const metadata = {
+            name: fileName, 
+            mimeType: 'application/zip', 
+        };
+        
+        const formData = new FormData();
+        formData.append('metadata', new Blob([JSON.stringify(metadata)], { type: 'application/json' }));
+        formData.append('file', blob);
+    
+        try {
+            const response = await axios.post(
+                'https://www.googleapis.com/upload/drive/v3/files?uploadType=multipart',
+                formData,
+                {
+                    headers: {
+                        Authorization: `Bearer ${accessToken}`,
+                        'Content-Type': 'application/octet-stream',
+                    },
+                }
+            );
+            alert('File uploaded successfully on google drive');
+        } catch (error) {
+            console.error('Error uploading file:', error);
+        }
+    }
+    
```

Comment:
ideally, this would be under services/CommunicationService.ts because this is where all axios stuff happens
alternatively it can be a new file und services like services/GoogleApiService.ts or so